helm/cluster-cloud-director/templates/_helpers.tpl

```diff
@@ -110,19 +122,24 @@ joinConfiguration:
     kubeletExtraArgs:
       {{- include "kubeletExtraArgs" . | nindent  6}}
       node-labels: "giantswarm.io/node-pool={{ .pool.name }}"
-{{- if $.Values.proxy.enabled }}
 files:
+{{- if $.Values.proxy.enabled }}
```

Comment:
I'm not sure what moving the `files` key outside of the if statement will do if there are no files to create (no static routes or proxy config) - i don't know if the template will be valid.

Reply:
so it's fine to have an empty `files:` key?

Reply:
Haven't tested with files but it's fine with other fields (e.g. postkubeadmscript). We'd have to test it in a non-proxied environment to make sure